tests/components/unifi/test_init.py

```diff
@@ -173,31 +174,68 @@ async def test_remove_config_entry_device(
         devices_response=[device_1],
     )
 
-    integration = await loader.async_get_integration(hass, config_entry.domain)
-    component = await integration.async_get_component()
-
-    # Remove a client
-    mock_unifi_websocket(message=MessageKey.CLIENT_REMOVED, data=[client_2])
+    assert await async_setup_component(hass, "config", {})
     await hass.async_block_till_done()
 
-    # Try to remove an active client: not allowed
+    ws_client = await hass_ws_client(hass)
+
+    # Try to remove an active client from UI: not allowed
     device_entry = device_registry.async_get_device(
         connections={(dr.CONNECTION_NETWORK_MAC, client_1["mac"])}
     )
-    assert not await component.async_remove_config_entry_device(
-        hass, config_entry, device_entry
+    await ws_client.send_json(
```

Comment:
We have a test helper for this now.

Example:

https://github.com/home-assistant/core/blob/48b167807503df5bd0d0b1b04681346e49e2f760/tests/components/config/test_device_registry.py#L281

Reply:
And a brand new one I see !
Thanks, should be good now